gradle.properties

```diff
@@ -4,20 +4,20 @@ max_errors=400
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop
-minecraft_version=1.20
-yarn_mappings=1.20+build.1
-loader_version=0.14.21
+minecraft_version=1.20.2
+yarn_mappings=1.20.2+build.4
+loader_version=0.14.23
 
 #Fabric api
-fabric_version=0.83.0+1.20
+fabric_version=0.90.0+1.20.2
 
 # Mod Properties
-mod_version = 1.8.4
+mod_version = 1.9.0
```

Comment:
Versioning here is wrong, as the previous 1.19.x build is 1.9.4...
My best proposal to fix this mess is to bump this to 1.10.0 or 1.9.5 (as my previous commit changed Worthiness behavior) and adopt semver build metadata to it, I.E. `1.10.0+1.19.2`. This keeps the version consistent across Minecraft versions

Reply:
Agreed.